code/src/components/TodoList.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import Checkbox from "@material-ui/core/Checkbox";
+
+import todos from "../reducers/todos";
+import "./TodoListStyle.css";
+
+const TodoList = () => {
+  const dispatch = useDispatch();
+  const items = useSelector((store) => store.todos.items);
+
+  return (
+    <div>
+      {items.map((todo) => (
+        <div className="todo-list-item-container" key={todo.id}>
+          <div className="todo-list-item-section">
+            <div className="todo-list-item-content ">
+              <Checkbox
+                color="primary"
+                checked={todo.isComplete}
+                onChange={() => {
+                  dispatch(todos.actions.toggleComplete(todo.id));
+                }}
+              />
+
+              <p
+                className="todo-list-item-text"
+                onClick={() => {
+                  dispatch(todos.actions.toggleComplete(todo.id));
+                }}
```

Comment:
The toggle works like a charm, nice work! 👍 One thing to think about if you want to make it a bit more user-friendly. If I write a short text, like "öl"  for example, it gets a bit tricky to toggle beween complete and not complete because the toggle is on the word and the word is very short and small.  To make it a bit easier for the user you could do so the entire section is "clickable",